fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -527,10 +527,12 @@ struct TagPartitionedLogSystem final : ILogSystem, ReferenceCounted<TagPartition
 	}
 
 	ACTOR static Future<TLogCommitReply> recordPushMetrics(Reference<ConnectionResetInfo> self,
+	                                                       Reference<Histogram> dist,
 	                                                       NetworkAddress addr,
 	                                                       Future<TLogCommitReply> in) {
 		state double startTime = now();
 		TLogCommitReply t = wait(in);
+		dist->sampleSeconds(now() - startTime);
```

Comment:
Move this to just before `return`.

There is also a `wait` inside `pushResetChecker` below that we may want to account for.

Reply:
Done.